src/main/java/com/durov/maks/cinema/service/ShoppingCartService.java

```diff
@@ -10,4 +10,6 @@
     ShoppingCart getByUser(User user);
 
     void registerNewShoppingCart(User user);
+
+    void deleteAllTickets(ShoppingCart shoppingCart);
```

Comment:
```suggestion
    void clear(ShoppingCart shoppingCart);
```
with your name somebody may be confused that it's gonna delete all tickets from database

Reply:
agree, I didnt think about it